unstructured/documents/coordinates.py

```diff
@@ -68,6 +68,19 @@ def convert_coordinates_to_new_system(
         rel_x, rel_y = self.convert_to_relative(x, y)
         return new_system.convert_from_relative(rel_x, rel_y)
 
+    def convert_multiple_coordinates_to_new_system(
```

Comment:
nit (aka as a followup ticket): this multiple conversion is just calling single conversion in serial. Looking at the code it seems we might be able to vectorize the operation to improve performance (memory and speed actually; you'd be surprised how much more memory python list takes vs. numpy array).

Reply:
I'll create a ticket for this then.